src/Parser.zig

```diff
@@ -813,6 +816,27 @@ pub const DeclSpec = struct {
     }
 };
 
+fn typeof(p: *Parser) Error!?Type {
+    if (p.eatToken(.keyword_typeof) orelse p.eatToken(.keyword_typeof1) orelse p.eatToken(.keyword_typeof2)) |_| {
```

Comment:
This would probably be better expressed with a switch on `tok_ids` and early return.